components/Toast.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import { useState, useEffect } from "react";
+import Sheet from "react-modal-sheet";
+
+import { useToastStore } from "@/lib/store";
+
+export default function Toast() {
+  const [toastDetails, showToast, setShowToast] = useToastStore((state) => [
+    state,
```

Comment:
![image](https://github.com/Glo-Foundation/glo-wallet/assets/6277878/13eeb96c-edd7-4c93-830d-322559e5c3bd)

This is a bit weird, either take state object or unpack its params, not both :P